config/routes.rb

```diff
@@ -0,0 +1,62 @@
+Rails.application.routes.draw do
+  get 'users/create'
+
+  get 'users/new'
+
```

Comment:
Review `get` routes on line 2 and 4 - they may have been added by a Rails generator but may not be needed. If they are needed, prefer using `resources :users, only: [...]` instead as the `resources ...`-routes tend to be a bit easier to work with than the `get ...`-routes (for example the `*_path` helper naming tends to be better with `resources ...`-routes).